src/libs/fileDownload/index.js

```diff
@@ -0,0 +1,39 @@
+import openURLInNewTab from '../openURLInNewTab';
+import getAttachmentName from './getAttachmentName';
+
+/**
+ * Downloading attachment in web, desktop
+ * @param {String} url
+ */
+export default function fileDownload(url) {
+    fetch(url)
+        .then(response => response.blob())
+        .then((blob) => {
+            // Create blob link to download
+            const href = URL.createObjectURL(
+                new Blob([blob]),
+            );
+
+            // creating anchor tag to initiate download
+            const link = document.createElement('a');
+
+            // adding href to anchor
+            link.href = href;
```

Comment:
I think we should add `link.style.display = 'none'` like in the davidwalshblog to make sure this anchor tag doesn't ever actually render.

Reply:
Okay, I will set this.

Reply:
Added the change